tutorials/03-connect-with-chipi/README.md

```
@@ -0,0 +1,107 @@
+// sign typed data (SNIP-12)
+const sig = await account.signMessage(typedData);
+
+// gasless execute
+const { transaction_hash } = await account.execute([
+  { contractAddress: USDC, entrypoint: "approve", calldata: [spender, "0x0", "0x0"] },
+]);
+```
+
+## How it works
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)